File: Remove Duplicates from Sorted Array/Remove Duplicates from Sorted Array.java
public class Solution {
    public int removeDuplicates(int[] A) {
        int newl = 0;
        for(int i = 0; i < A.length; i++) {
            if (newl == 0 || A[i] != A[i-1]) { //encounter a new element
                A[newl++] = A[i];
            }
        }
        return newl;
    }
}

File: Remove Duplicates from Sorted Array/Remove Duplicates from Sorted Array.py
class Solution:
    # @param a list of integers
    # @return an integer
    def removeDuplicates(self, A):
        newl=0
        for i in range(len(A)):
            if newl==0 or A[i]!=A[i-1]:
                A[newl]=A[i]
                newl+=1
        return newl
